src/components/alert/alertCenter.svelte

```diff
@@ -53,19 +63,28 @@
     }
 
     resumeDismiss() {
-      if (!this.duration) return
+      if (!this.duration || this.actions.length) return
```

Comment:
@fallaciousreasoning, do you agree with this?

Reply:
Hmmm, I don't think so - if you set a duration it should hide then - you can set a zero duration with actions.

Probably doesn't work in the story because the duration is a string not a number. Does 

```suggestion
      if (!+this.duration) return
```

work?

Reply:
Sorry, the original worked, I was just thinking that it doesn't make much sense to have a timeout if there's an action.

Reply:
I think it still does - for example, on Android we show an Alert with an `Undo` action after closing a tab for 3 seconds.